setup.py

```diff
@@ -0,0 +1,97 @@
+"""
+This module defines the attributes of the
+PyPI package for the mbed SDK test suite
+"""
+
+"""
+mbed SDK
+Copyright (c) 2011-2021 ARM Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Author: Przemyslaw Wirkus <Przemyslaw.Wirkus@arm.com>
+"""
+
+import os
+from io import open
+from distutils.core import setup
+from setuptools import find_packages
+
+DESCRIPTION = "mbed 3.0 onwards test suite, codename Greentea. The test suite is a collection of tools that enable automated testing on mbed-enabled platforms"
+OWNER_NAMES = 'Anna Bridge, Qinghao Shi'
```

Comment:
Should we be tying these packages to individuals? We should say something like "The ARMmbed team" for the package owner/maintainer.

Reply:
Just followed like [Mbed tools]( https://github.com/ARMmbed/mbed-tools/blob/master/setup.py#L20)